collect_app/src/main/java/org/odk/collect/android/preferences/keys/ProjectKeys.kt

```diff
@@ -66,6 +66,9 @@ object ProjectKeys {
     const val KEY_BACKGROUND_LOCATION = "background_location"
     const val KEY_BACKGROUND_RECORDING = "background_recording"
 
+    // experimental_preferences.xml
+    const val KEY_EXPERIMENTAL_GEOPOINT = "experimental_geopoint"
```

Comment:
We might want to delete the old geopoint stuff now and move to always using the new experience, but I'd be happy to do a beta with it as "opt in" to get more targeted feedback for the moment.